### DISCUSSION TITLE
Project roadmap, community support, stability

### DISCUSSION THREAD

**Discussion by etabackman** [jnsmalm/pixi3d#8] [Q&A]
I have a working animation engine developed using PixiJS. It is used for creating motion graphics, so it mostly uses text, images, video, and sprites. I have an editor that runs in React Native to allow the user to define the hierarchy of elements, and then the animation rendered to a static video in the cloud using Puppeteer.

I am interested in adding 3D capabilities, such as 3D text, lighting, 3D transforms...

What is the roadmap for this project, how stable is it, how many people are working on it?

**Comment by jnsmalm** [comment]
I am the only developer for this project and right now I'm working towards a v1.0, which should not be that far off. But for this to happen some things are needed:

- Enough people try it and give feedback about functionality and API.
- At least one project running it in production used by thousands of people. I have used Pixi3D in this kind of "real" project, which will be put in a live environment in a couple of months.

I want to keep the all the main features in there right now (almost all features in there at this point) stable, meaning I don't want to change the API for those. I really enjoy working on this project (about a year now) and will probably be able to fix most bugs pretty quickly when you find them.

Did you try it yet?

**Comment by jnsmalm** [comment]
This library is now being used in production in different projects with hundres of thousand of users. Have not received any issues at all yet. This library is definitely production ready now.
